### PR TITLE
Replaces all Kubecost domain links with relative links

### PR DESCRIPTION
**Pull request by bstuder99**
## Proposed Changes

Replaces all examples of "docs.kubecost.com" with relative links to the Markdown files.

### CODE REVIEW

File: apis/apis-overview/allocation-trends-api.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Analyzes change in total cost of allocations relative to a previous window of th
 {% endswagger-description %}
 
 {% swagger-parameter in="path" name="window" required="true" type="string" %}
-Duration of time over which to query. Compares cost usage of window to cost usage window of equal size directly preceding it. Accepts all standard Kubecost window formats (See our doc on using [the `window` parameter](https://docs.kubecost.com/apis/apis-overview/assets-api#using-window-parameter)).
+Duration of time over which to query. Compares cost usage of window to cost usage window of equal size directly preceding it. Accepts all standard Kubecost window formats (See our doc on using [the `window` parameter]/apis/apis-overview/assets-api.md#using-window-parameter)).
```
Comment (srpomeroy) [inline apis/apis-overview/allocation-trends-api.md:9]: Missing opening parenthesis

```suggestion
Duration of time over which to query. Compares cost usage of window to cost usage window of equal size directly preceding it. Accepts all standard Kubecost window formats (See our doc on using [the `window` parameter](/apis/apis-overview/assets-api.md#using-window-parameter)).
```

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Nice job! I am honestly a bit surprised to see this many. I thought we fixed these months ago.

**Comment by bstuder99** [comment]
We've fixed a lot of them over time but I don't think I ever did a sweep through all content to completely remove them

**Comment by bstuder99** [comment]
Broken link can be ignored, more Thanos Helm stuff I'm going to address in a separate PR

**Review by srpomeroy — APPROVED** [review]

**Pull request merged**